db/migrate/20160203145841_add_content_id_to_about_page.rb

```diff
@@ -0,0 +1,16 @@
+require "securerandom"
+
+class AddContentIdToAboutPage < ActiveRecord::Migration
+  def up
+    add_column :about_pages, :content_id, :string
+
+    AboutPage.reset_column_information
+    AboutPage.all.each do |about_page|
+      about_page.update_attribute(:content_id, SecureRandom.uuid)
```

Comment:
@jamiecobbett Sorry for any confusion - I didn't mean to describe capital-A atomicity in the ACID sense - more as one single operation in a conventional migration. Your first paragraph makes sense to me as general guidance. :+1: